src/chickens.c

```diff
@@ -91,6 +94,7 @@ const struct midr_part_features features_m2 = {
 // XXX figure out what features are actually available on M4
 const struct midr_part_features features_m4 = {
     .sleep_mode = SLEEP_NONE, // XXX probably new mode required
+    .uncore_version = UNCORE_V2,
```

Comment:
I'd probably just leave this out for now since we have no idea what M4 actually supports (and because M4 isn't really supported by any means for now)